infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/AsyncStorageModeValidator.java

```diff
@@ -70,8 +72,12 @@ public AsyncStorageModeValidator(
 
     this.pvcStrategy = pvcStrategy;
     this.allowUserDefinedNamespaces = allowUserDefinedNamespaces;
-    this.defaultNamespaceName = defaultNamespaceName;
     this.runtimesPerUser = runtimesPerUser;
+
+    this.isPvcStrategyNotValid = !COMMON_STRATEGY.equals(pvcStrategy);
+    this.runtimesPerUserLimited = runtimesPerUser > 1;
```

Comment:
| configured | condition | runtimesPerUserLimited |
| ---------------|-------------|----------------------------------|
| runtimes unlimited | -1 > 1 |  false | 
|  1 limit for runtimes |  1 > 1 |  false | 
|  2 limit for runtimes |  2 > 1 |  true | 
|  100 limit for runtimes |  100 > 1 | true | 

^ is it right? If yes - I suppose something like `singleRuntimeAllowed` would be better as well, as `runtimesPerUser == 1` condition.

Reply:
hm, probably yes it will be better, for understanding